armotypes/pagination_structs.go

```diff
@@ -0,0 +1,180 @@
+package armotypes
```

Comment:
is armo types the right package ?
if we are already breaking it to packages these are not model types but API types

Reply:
I am not sure about the division of types<>APIs, but I thought that APIs are for internal stuff, while armotypes is for external as well